antha-lang/antha
# Generate found test data for tests.

It's necessary to add test data to files, however it is not trivial
to generate the test data.
By always exporting the test data, it becomes easy to create and
update workflow tests.

Test Plan:
Run standard workflow tests:
  * Check that we do not affect test times (through extra processing)
  * Check that we can easily update test bundles.
